Dealer.h

```diff
@@ -0,0 +1,47 @@
+#ifndef DEALER_H
+#define DEALER_H
+#include "Deck.h"
+#include <iostream>
+#include <vector>
+#include <string>
+#include <string_view>
+class Dealer
+{
+private:
+  Deck deck{};
+  std::string m_name{};
+  std::vector<std::string> Hand{};
+  int m_score{};
+  static const int c_maxScore{21};
+public:
+  Dealer(std::string_view name)
+    : m_name{name}
+  {
+  }
+
+  friend std::ostream& operator<<(std::ostream& out, const Dealer& d)
```

Comment:
Let's put every implementation in the cpp file. Writing code in the header file makes development slightly easier at the cost of making reading the header file harder and slowing down compiler times (It also _can_ speed up your code by allowing the compiler to inline the function more easily. Don't think about that, I only mention it to be complete). Since we read code a _lot_ more often then we write code it's better to prioritize the reading, not the writing